docs/source/optimization_ov.mdx

```diff
@@ -62,6 +62,36 @@ tokenizer.save_pretrained(save_dir)
 
 The `quantize()` method applies post-training static quantization and export the resulting quantized model to the OpenVINO Intermediate Representation (IR). The resulting graph is represented with two files: an XML file describing the network topology and a binary file describing the weights. The resulting model can be run on any target Intel device.
 
+##  Weight-only quantization
+
+You can optimize the performance of text-generation LLMs by quantizing weights to various precisions that provide different performance-accuracy trade-offs.
+
+```python
+from optimum.intel import OVModelForCausalLM
+
+model = OVModelForCausalLM.from_pretrained(model_id, load_in_8bit=True)
+```
+
+> **NOTE:** `load_in_8bit` is enabled by default for models larger than 1 billion parameters.
+
+For the 4-bit weight quantization we recommend using the NNCF API like below:
```

Comment:
I would much recommend optimum-cli over NNCF API for this. It's such a quick and easy method. And (unless that has been fixed very recently) NNCF fails on SPR/EMR with a BF16 error and it's not easy to know how to work around that.

Reply:
I guess we fixed this issue in the recent version of NNCF. @alexsu52, please confirm.

Reply:
Yes, you are right. Models with float16 and float32 weigths work on SPR/EMR.